docs/api-reference/reference.asciidoc

```diff
@@ -463,6 +463,7 @@ DiscoveryServiceSpec defines the desired state of DiscoveryService
 | *`enabledNamespaces`* __string array__ | EnabledNamespaces is a list of namespaces where the envoy discovery service is enabled. In order to be able to use marin3r from a given namespace its name needs to be included in this list because the operator needs to add some required resources in that namespace.
 | *`image`* __string__ | Image holds the image to use for the discovery service Deployment
 | *`debug`* __boolean__ | Debug enables debugging log level for the discovery service controllers. It is safe to use since secret data is never shown in the logs.
+| *`resources`* __link:https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.17/#resourcerequirements-v1-core[$$ResourceRequirements$$]__ | Resources holds the Resource Requirements to use for the discovery service Deployment. When not set it defaults to no resource requests nor limits. CPU and Memory resources are supported.
```

Comment:
Just out of curiosity, do you know why is it linking to the v1.17 k8s documentation specifically?

Reply:
Yes. It's behavior of crd-ref-docs binary.

Looking at it it seems it can be controlled in the crd-ref-docs configuration file.
In our case located in api-reference/config.yaml:
```
msoriano@localhost:~/go/src/github.com/miguelsorianod/marin3r/docs (add-envoy-webhooksidecar-configurable-resources)$ cat api-reference/config.yaml 
processor:
  # RE2 regular expressions describing type fields that should be excluded from the generated documentation.
  ignoreFields:
    - "TypeMeta$"

render:
  # Version of Kubernetes to use when generating links to Kubernetes API documentation.
  kubernetesVersion: 1.17
```

Notice the `.render.kubernetesVersion` field